lambdas/services/create_document_reference_service.py

```diff
@@ -141,21 +149,17 @@ def prepare_doc_object(
 
         logger.info(PROVIDED_DOCUMENT_SUPPORTED_MESSAGE)
 
-        if validated_doc.docType == SupportedDocumentTypes.LG.value:
+        if validated_doc.docType in [
+            SupportedDocumentTypes.LG.value,
+            SupportedDocumentTypes.ARF.value,
+        ]:
```

Comment:
Is this check necessary? if it is not one of those it would have thrown a ValidationError before.

Reply:
Thanks! Yes we don't need this check anymore. Will take this away
 Sorry I just combined the previous if-else branches and didn't notice this 😓